src/test/java/com/fasterxml/jackson/core/Base64VariantsTest.java

```diff
@@ -0,0 +1,31 @@
+package com.fasterxml.jackson.core;
```

Comment:
Could this be added to one of existing base64 tests, instead of a new test class?